runtime/domain.c

```diff
@@ -398,8 +438,25 @@ int caml_reallocate_minor_heap(asize_t wsize)
       (void*)domain_self->minor_heap_area,
       (char*)domain_state->young_end - (char*)domain_state->young_start);
```

Comment:
Reading this comment https://github.com/ocaml/ocaml/pull/11082/files#diff-67115925103982a8ebeb085cfab5ef31a182c9a442bc51e053934364d3750dafR383-R387: 

```suggestion
      (char*)domain_state->minor_heap_wsz);
```